<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>808</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel#label_top{
background-color:rgb(0, 170, 255)
}
QPushButton#btn_min{
	border:none;
	image: url(:/res/zuixiaohua.png);
}
QPushButton#btn_add{
	border:none;
	image: url(:/res/search.png);
}
QLabel#label_title{
	border:none;
}
QPushButton#btn_close{
	border:none;
}
QPushButton#btn_min:hover{
	background-color: rgba(190 ,190 ,190 ,100);
	
}
QPushButton#btn_close:hover{
	background-color: rgb(255, 0, 0)
}
QPushButton#btn_more{
	border:none;
	
	image: url(:/res/more.png);
}
QPushButton#btn_more:hover{
	background-color: rgba(190 ,190 ,190 ,100);
}
QLineEdit#lineEdit_bardian{
	border:none;
	background-color: rgba(255, 255, 255,0);	
	font:200 15px ;
	color: rgb(247, 247, 247);
}
QLineEdit#lineEdit_add{
	border:none;	
	font:200 15px ;
}
QTabBar::tab{height:30}
QTabBar::tab{width:153.33333333333333333}
QTabWidget#tabWidget QTabBar::tab{
	font:200 17px;
	color: rgb(171, 171, 171);
	background-color:transparent;
}
QTabWidget#tabWidget QTabBar::tab::hover{
	font:200 17px;
	color: rgb(47, 47, 47);
}
QTabWidget#tabWidget QTabBar::tab:selected{
	color: rgb(0, 170, 255);
}
.QTabWidget#tabWidget {
	border:none;
	background-color:transparent;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>461</width>
     <height>801</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>460</width>
      <height>614</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="title">
      <string>消息</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="minimumSize">
         <size>
          <width>460</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="title">
      <string>联系人</string>
     </attribute>
     <widget class="QToolBox" name="toolBox_Person">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>460</width>
        <height>582</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabSpacing">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>好友</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_4" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>460</width>
         <height>513</height>
        </rect>
       </property>
       <attribute name="label">
        <string>朋友</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_5" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>家人</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_6" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="title">
      <string>群聊</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>462</width>
      <height>155</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>120</height>
     </size>
    </property>
    <widget class="QLabel" name="label_top">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>462</width>
       <height>155</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_head">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>60</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/head.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/mainTitle.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/weather.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="username">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bardian">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>编辑个性签名</string>
     </property>
    </widget>
    <widget class="QLabel" name="link_state">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>125</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_more">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>-1</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/guanbi.png</normaloff>:/res/guanbi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_min">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>6</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/zuixiaohua.png</normaloff>:/res/zuixiaohua.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_add">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>152</y>
      <width>460</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>输入账号添加好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>154</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
